src/ensembl/src/shared/toggle-image-button/ToggleImageButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import dafaultStyles from './ToggleImageButton.scss';
+import ImageHolder from './ImageHolder';
+
+export enum ImageButtonStatus {
+  ACTIVE = 'active',
+  INACTIVE = 'inactive',
+  DISABLED = 'disabled'
+}
+
+type Props = {
+  buttonStatus: ImageButtonStatus;
+  description?: string;
+  image: any;
+  imageStyles?: any;
+  onClick: () => void;
+};
+
+const ToggleImageButton = (props: Props) => {
+  return (
+    <div
+      onClick={
+        props.buttonStatus === ImageButtonStatus.DISABLED
+          ? () => {
+              return;
+            }
+          : () => props.onClick()
```

Comment:
Perhaps instead of writing this in jsx itself, we could have something like:

```
const buttonProps = props.buttonStatus === ImageButtonStatus.DISABLED ? {} : { onClick: props.onClick }

<div {...buttonProps}>
  blah
</div>
```

Reply:
That's done.